Makefile

```diff
@@ -85,11 +95,24 @@ clean-frontend-install:
 	@printf "$(FORMATBLUE)$(FORMATBOLD)---> Cleaning frontend install <---$(FORMATRESET)\n"
 	@echo ""
 	rm -Rf $(FRONTEND_DIR)/node_modules
+	rm -Rf $(FRONTEND_DIR)/yarn.lock
 .PHONY: clean-frontend-install
 
-clean: clean-var clean-backend-install clean-frontend-install clean-frontend-build clean-pycache
+clean: clean-backend-install clean-db clean-frontend-install clean-frontend-build clean-pycache
 .PHONY: clean
 
+clean-pycheck:
+	@echo ""
+	@printf "$(FORMATBLUE)$(FORMATBOLD)---> Cleaning Pycheck installation <---$(FORMATRESET)\n"
+	@echo ""
+	npm uninstall -g pycheck
+
+install-pycheck:
+	@echo ""
+	@printf "$(FORMATBLUE)$(FORMATBOLD)---> Installing Pycheck <---$(FORMATRESET)\n"
+	@echo ""
+	npm install -g pycheck
```

Comment:
how about yarn users (like me)? this would be `yarn add` instead of `npm install` for those

maybe we could use a variable to set the package manager in the makefile (yarn or npm)?